tests/test_data.py

```diff
@@ -172,8 +172,8 @@ def test_v3_power_model_parsing():
         )
         data.load_data(topics.PowerModelV3.plan_name, dumps(plan_name_data))
 
-        assert data["rec_power"] == 10
-        assert data["rec_speed"] == 20
-        assert data["zdist"] == 30
-        assert data["predicted_max_speed"] == 100
-        assert data["plan_name"] == "default.pkl"
+        assert data["rec_power"].get() == 10
+        assert data["rec_speed"].get() == 20
+        assert data["zdist"].get() == 30
+        assert data["predicted_max_speed"].get() == 100
+        assert data["plan_name"].get() == "default.pkl"
```

Comment:
Can you add some `DataValue` tests that test things like
- Initially, `is_valid` is `False` and `get*` return `None`
- After calling `update`, the above functions return correctly
- After waiting 5 seconds, the above functions (or ideally less, just so the tests run faster, if you add something to set a custom data expiry duration) return `False`/`None` again

and in a seperate test,
- Check that the formatting in `get_string` works as expected